src/app/repositories/repositories.module.ts

```diff
@@ -0,0 +1,15 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { RepositoriesRoutingModule } from './repositories-routing.module';
+import { RepoPageComponent } from './repo-page/repo-page.component';
+
+
```

Comment:
extra blank line

Reply:
This file was generated with Angular-cli. Also, I think 2 spaces is also acceptable here. @zvonimirfras tagging you in case I'm wrong

Reply:
It should just be one blank line 👍

Reply:
We had a lot of places that separate imports from the code with two empty lines. We can standardize on one. Looks like you guys are leaning towards always having one, so let's do that.

Reply:
I'm pretty sure it's always been one blank +comment if needed?